tutorverse-server/index.js

```diff
@@ -42,12 +42,17 @@ socketIO.on('connection', (socket) => {
   }
 
   //Incoming messages
-  socket.on('sendMessage',(message) => {
+  socket.on('sendMessage',async (message) => {
     const {fromId, toId, content} = message;
     console.log(message)
     const socketIDs = userSocketIds[toId]
-    console.log("Message sent!" ,content,toId)
-    console.log(socketIDs)
+    
+    await Message.create({
```

Comment:
So using await here forces this db creation operation to finish BEFORE moving onto the next bit of code here, which slows us down from actually sending the message to the user. We can probably remove the `await` here and just let the promise resolve on its own (maybe log errors if it fails). 
 
 Then, we can continue sending the message without waiting for the db operation to finish.